File: 80_remove_duplicates_from_sorted_array_2/removeDuplicates.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        idx = 0

        for i in range(len(nums)):
            if idx < 2 or nums[i] != nums[idx - 2]:
                nums[idx] = nums[i]
                idx += 1
        return idx

File: TopInterview150/80_remove_duplicates_from_sorted_array_2/removeDuplicates.ts
function removeDuplicates(nums: number[]): number {
  let pos = 0;

  for (let i of nums) {
    if (pos < 2 || i != nums[pos - 2]) {
      nums[pos] = i;
      pos++;
    }
  }

  return pos;
}
